### PR TITLE
Don't allow orphaned profiles to show up in the Default Profiles

### PR DESCRIPTION
**Pull request by DHowett**

### CODE REVIEW

**Comment by DHowett** [comment]
This is already part of #18206

**Comment by Vintagor01** [comment]
Check out

File: src/cascadia/TerminalSettingsEditor/ProfileViewModel.idl [hunk 1]
```diff
@@ -84,7 +84,7 @@ namespace Microsoft.Terminal.Settings.Editor
         void CreateUnfocusedAppearance();
         void DeleteUnfocusedAppearance();
 
-	Boolean Orphaned { get; };
+        Boolean Orphaned { get; };
```
Comment (DHowett) [inline src/cascadia/TerminalSettingsEditor/ProfileViewModel.idl:87]: yeah i tabs'd

**Review by lhecker — APPROVED** [review]

**Review by carlos-zamora — APPROVED** [review]

**Pull request merged**